setup.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from setuptools import setup, find_packages
+from os import path
+
+
+UTILS_CV = "utils_cv"  # Utility folder name
+# UTILS_CV_PATH = path.join(path.abspath(path.dirname(__file__)), UTILS_CV)
+# README = path.join(UTILS_CV_PATH, "README.md")
+README = path.join(UTILS_CV, "README.md")
+exec(open(path.join(UTILS_CV, "__init__.py")).read())
+
+with open(README, encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name="utils_cv",
+    version=__version__,
+    description="Computer Vision Utilities",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/microsoft/ComputerVision",
+    project_urls={
+        "Bug Tracker": "https://github.com/microsoft/ComputerVision/issues",
+        "Source Code": "https://github.com/microsoft/ComputerVision",
+        "Documentation": "https://github.com/microsoft/ComputerVision",
+    },
+    author="CVDev Team at Microsoft",
+    classifiers=[
+        "Development Status :: 2-Pre-Alpha",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+    ],
+    include_package_data=True,
+    # Not fully test the dependency coverage, please make sure to sync
+    # with environment.yml
```

Comment:
Rather than explicitely listing all dependencies here, is it possible to refer to the environment.yml file? That way we also make sure that these two places stay in sync (also because the environment.yml file is used in our tests)

Reply:
instead of using our enviornment.yml, it _might_ be best to have the environment.yml use the utils_cv package, as so:

```yml
# environment.yml
name: cv
channels:
- defaults
...
dependencies:
- python==3.6.8
- pytorch==1.0.0
- torchvision>=0.3.0
- fastai==1.0.48
- ..
- pip:
  - azureml-sdk[notebooks,contrib]>=1.0.30
  - utils_cv>=0.0.1 <-- [install our lib here]
  - ...
```

Reply:
I agree to add `utils_cv` in environment.yml.  That would make sense.

Reply:
@PatrickBue @jiata See 63daec9db29f3f64b9e804afb4ac091b2552ff91
https://github.com/microsoft/ComputerVision/blob/63daec9db29f3f64b9e804afb4ac091b2552ff91/environment.yml#L12-L28

Reply:
Please update/revert this setup script to reflect that the environment.yml file will stay unchanged.

Reply:
Reverted.